migrations/1528746007_stream_items.up.sql

```diff
@@ -0,0 +1,16 @@
+--
+-- version: 1528746007
+--
+
+BEGIN;
+
+CREATE TABLE stream_items (
+    item_id       BIGINT       PRIMARY KEY,
+    stream_id     BIGINT       NOT NULL,
+    entity_id     BIGINT       NOT NULL
+);
+
+CREATE UNIQUE INDEX stream_items_stream_entity_id ON stream_items (stream_id, entity_id);
```

Comment:
No need to name these indices; I've only been naming the JSONB ones because they don't get useful names if they contain JSONB queries.